cli/cmd/install.go

```diff
@@ -358,11 +363,34 @@ func renderCRDs(w io.Writer, options valuespkg.Options) error {
 		return err
 	}
 
-	_, err = w.Write(buf.Bytes())
-	return err
+	if format == "json" {
```

Comment:
In `cli/cmd/root.go` we have a `jsonOutput` constant. Should we use that here?

https://github.com/linkerd/linkerd2/blob/b028db78a2f0d33b1424adf62bc50cab2e48916e/cli/cmd/root.go#L26-L28

should we also add one for `yaml`?